source/beamline/BioXAS/BioXASSideBeamline.cpp

```diff
@@ -208,7 +208,7 @@ void BioXASSideBeamline::setupComponents()
 
 	// Endstation safety shutter.
 
-	endstationShutter_ = new  BioXASEndstationShutter("BioXASSideEndstationShutter", "SSH1607-5-I22-01", this);
+	endstationShutter_ = new  CLSExclusiveStatesControl("SSH1607-5-I22-01", "SSH1607-5-I22-01:state", "SSH1607-5-I22-01:opr:open", "SSH1607-5-I22-01:opr:close", this);
```

Comment:
'BioXASShutters' is also a general class that is meant to be used equally well for Side, Main, and Imaging. All three have the same front-end shutters, but the endstation shutter is different for each. We could just as easily have 'BioXASSideShutters', 'BioXASMainShutters', and 'BioXASImagingShutters' where each shutter set is defined (instead of in each beamline class). I just opted for the 'single class' option.